### ISSUE TITLE
Home - Download (top right hand side) does not seem to give 'Dataset in CLDF' ?

### ISSUE THREAD

**Issue by klamermaf** [Anaphory/lexibank#26]

**Comment by Anaphory** [comment]
True! Where the other downloads are generated automatically, the “Dataset in CLDF” Download is a ZIP file of all our data. I'll put it there when we have finished all data editing for a moment.

**Comment by OwenAmarasi** [comment]
This still seems to be an issue. The top right hand side has text:

Lexirumah%202.1.1_dataset.cldf.zip
    Dataset in CLDF 

But nothing to download.
